components/filter-components/MapFilters.vue

```diff
@@ -217,6 +218,7 @@
             </div>
           </div>
         </div>
+        </div>
```

Comment:
I don't think this </div> is needed and think this is why my page won't compile. I can't see the pages properly when I run npm run dev. Think removing the extra closing div should hopefully fix.